### ISSUE TITLE
A website for this project

### ISSUE THREAD

**Issue by strypey** [TheAssassin/fetcher#2]
Once we confirm our final name, we need to get a website up and running. I've done the 'Learn Enough Git to be Dangerous' tutorial, which walked me through creating and maintaining GH sites, and the same steps pretty much work on GitLab. Shall we:
* set up a GH organisation so we can have a separate repo for the homepage along with the repo for the app itself?
**OR**
* host the Codename homepage on GitLab?

Or is it possible to make the homepage part of the app repo?

**Comment by TheAssassin** [comment]
I can host a homepage on my servers, that's not a problem. Just need a few bucks for a domain.

I'd set up a GitHub organization anyway because that allows me to invite you as an owner. Otherwise, as long as the project is officially "mine", you can't get this level of permissions.

We should first decide on a name, though, and then think about further steps.

**Comment by strypey** [comment]
On 2018-04-27 14:21, TheAssassin wrote:
> I can host a homepage on my servers, that's not a problem.

I'm fine with that. But since we're using GH anyway, it would make it
easier to recruit a volunteer to be a webmaster, if working on it only
requires altering a GH repo.

> Just need a few bucks for a domain.

I'm happy to stump up for that. Once we are up and running, I could ask
for donations on the Trisquel forums. I'm hoping our fork will become
the officially recommended app for making boot media on the Trisquel
wiki.

> I'd set up a GitHub organization anyway because that allows me to invite you as an owner ... We should first decide on a name, though, and then think about further steps.

Makes sense. Did you see the email on naming I send you today, with some
ideas about trying to use 'image'?

**Comment by TheAssassin** [comment]
Hosting a website != developing on a website. I am hosting a lot of websites whose source code is on GitHub, e.g., https://newpipe.schabi.org (https://github.com/TeamNewPipe/website/) or https://appimage.org (https://github.com/AppImage/AppImageKit/tree/gh-pages).

I didn't try to say I need to collect money to rent a domain, I'll happily sponsor it.

I saw your mail, I'll reply later. In any case, I'm fine with your non-"image" suggestion.

**Comment by strypey** [comment]
On 2018-04-27 19:26, TheAssassin wrote:
> Hosting a website != developing on a website.

Ok. Do you mean we'd have a repo for the website on GH, under our GH
organisation, and you'd deploy to your server from that repo? Sounds
fine to me.
 
> I didn't try to say I need to collect money to rent a domain, I'll happily sponsor it.

That's good of you. Just trying to make sure I'm pulling my weight ;)

**Comment by strypey** [comment]
OK, so now we've got our final name; Fetcher. Once you've set up the organization, I will create a repo for a simple homepage.

Next step is to get a domain name. Fetcher is a common word so most of the .com .org etc are already taken, but we could either use a more obscure extension (www.fetcher.gratis?), or something like fetcher-app.org?

**Comment by TheAssassin** [comment]
https://github.com/fetcher is already taken. We should decide on a domain name, and use that for the organization as well to provide some consistent experience.

**Comment by strypey** [comment]
In reply to your comment in #4 , the website for me is not mainly for promotion of the app as a product, but for promoting the project we have undertaken, and encouraging more people to contribute (including graphic designers). I was just thinking that working on images for the app (as mentioned in #4 ) may be more attractive to a graphic designer if they can do a consistent design across the whole project (homepage included), and put that on their CV and in their portfolio etc.

**Comment by strypey** [comment]
Sorry I've left this hanging for so long. Are you still working on it? It would be great to have it ready to promote when [Trisquel 9 is released](https://trisquel.info/en/forum/trisquel-9-ready-development-and-testing).

**Comment by strypey** [comment]
Sorry I've left this hanging for so long. Are you still working on it? It would be great to have it ready to promote when [Trisquel 9 is released](https://trisquel.info/en/forum/trisquel-9-ready-development-and-testing).

**Comment by TheAssassin** [comment]
I haven't invested any much more time in here since, I've been quite busy with other things, and this won't get better for the next weeks. Right now, I think we're best off presenting the project here on GitHub properly.
Edit: meant like, "set up domain and have it redirect here" or something like that.

I developed the idea of creating a better image writer in a real programming language like C++ last year, there's also libraries for this that you just need to plug together. It shall be as minimal as possible, and provide a look and feel similar to this app (but maybe less fancy, a tool for "engineers" I'd say, that just does one job and does it right). So this project is only temporary until I get to start hacking on that.

Please open issues and describe what needs to be done, and I'll try to find some time to work on those.

**Comment by strypey** [comment]
@TheAssassin:
> I developed the idea of creating a better image writer in a real programming language like C++ last year ... a tool for "engineers" 

This kind of exists (c++ with QT4 for cross-platform GUI), although
AFAIK it only makes boot USBs, not CDs/DVDs:
https://github.com/unetbootin/unetbootin

There are two reasons the Trisquel community doesn't like to recommend
it. One is that most of the distros it offers to download from within
the UI are not endorsed by the FSF. This is discussed here: 
https://trisquel.info/en/forum/unetbootin-fork

A few years ago someone made a fork that included only [FSF-endorsed
distros](https://www.gnu.org/distros/free-distros.html), but this
doesn't appear to have been maintained. Maybe because this guy created a
lot of extra work for himself, because every time a distro releases a
new version the program needs to be updated to download that instead of
the last version:
https://notabug.org/tomlukeywood/FreeNetBootin

A fork that only burns .ISOs already downloaded by the user (do one job
well) would be great. 

The other issue that used to come up in the Trisquel forums about
unNetbootin is that it hasn't always done a great job of making boot
USBs that work reliably for all distros. I haven't used it for a long
time, since I learned how to use dd in a terminal. YMMV ;)

**Comment by strypey** [comment]
I agree we need to be clear on what our project is before it's worth even thinking about a website. I've started a new issue to discuss that (#5).
